packages/office-ui-fabric-react/src/components/Panel/Panel.base.tsx

```diff
@@ -144,6 +144,7 @@ export class PanelBase extends BaseComponent<IPanelProps, IPanelState> implement
     const isLeft = type === PanelType.smallFixedNear || type === PanelType.customNear ? true : false;
     const isRTL = getRTL();
     const isOnRightSide = isRTL ? isLeft : !isLeft;
+    const isHeaderInContentInner = !!(this.props.onRenderNavigation || this.props.onRenderNavigationContent);
```

Comment:
how about we use a more semantic variable here. What we're really checking is if we have custom navigation or not. How about `hasCustomNavigation` or `navigationIsCustom`